SnackOverflow/src/test/resources/db/migration/V1__Initial_schema.sql

```diff
@@ -0,0 +1,94 @@
+CREATE TABLE IF NOT EXISTS`user` (
```

Comment:
Why do you need to duplicate the database migrations between the src and test folders?